### PR TITLE
build(deps): bumps Jacoco to 0.8.2

### PR DESCRIPTION
**Pull request by kifj**
#### Short description of what this resolves:
- update to Jacoco 0.8.2 works for JDK 10 and 11

#### Changes proposed in this pull request:

- Java 1.8 is minimum requirement
- Wildfly 14.0.0.Final is used for testing
- Arquillian 1.4.0.Final is used as base
- Jacoco is upgraded to 0.8.2 

**Fixes**: #
- works with Java 11

### CODE REVIEW

**Comment by bartoszmajsak** [comment]
Travis build fails with 

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:2.10:unpack (unpack) on project arquillian-jacoco: Unable to find artifact. Failure to find org.wildfly:wildfly-dist:zip:14.0.0.Final in http://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced
```

Mind having a look?

**Comment by kifj** [comment]
org.wildfly:wildfly-dist:zip:14.0.0.Final is not (yet) in http://repo.maven.apache.org/maven2, but I'm using the jboss releases repo. So it's either
- downgrade the version to 13.0.0.Final
- update the settings.xml
- wait

What do you suggest?

**Comment by bartoszmajsak** [comment]
If that's a matter of sync with Central, then we can wait till tomorrow. Many thanks for your contributions!

**Comment by kifj** [comment]
Since the repo is synced now, I have triggered a new build with an empty commit. Now the build is successful

File: pom.xml [hunk 1]
```diff
@@ -31,9 +31,9 @@
     <version.assertj>2.6.0</version.assertj>
 
     <!-- Overridden from parent -->
-    <jdk.min.version>1.5</jdk.min.version>
-    <maven.compiler.target>1.5</maven.compiler.target>
-    <maven.compiler.source>1.5</maven.compiler.source>
+    <jdk.min.version>1.8</jdk.min.version>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <maven.compiler.source>1.8</maven.compiler.source>
```
Comment (bartoszmajsak) [inline pom.xml]: What's the reason for updating to JDK8?

File: pom.xml [hunk 2]
```diff
@@ -210,9 +216,9 @@
       <id>remote</id>
       <dependencies>
         <dependency>
-          <groupId>org.wildfly</groupId>
+          <groupId>org.wildfly.arquillian</groupId>
           <artifactId>wildfly-arquillian-container-remote</artifactId>
-          <version>${version.wildfly}</version>
+          <version>2.1.1.Final</version>
```
Comment (bartoszmajsak) [inline pom.xml]: Maybe you could externalize it to `version.wildfly.arquillian`?

File: pom.xml (hunk 1 above)
Comment (kifj) [reply pom.xml]: I just reverted this, there was no technical reason for it.

**Comment by kifj** [comment]
I hope this fits your review

**Pull request merged**